cypress/support/step_definitions/validations.md

```diff
@@ -445,6 +445,31 @@ While validating, if a key is present in both fcs-validation jsons (eg: cypress/
         ]
 }
 
+# Dynamic Content Validation
```

Comment:
This is not understandable to a user how need to write their own validation. 

Currently, for validating device specific values, the validation object will have type "deviceContentValidation" . FCS will support this in two ways

- static data - explain about mac.json and where to add it
- dynamic data - explain about 

Readme should be written in terms of usage and how a user can use this feature. 

Background : 
Usage : 
Pre-requisites : 
Setup /configurations : 
How to test : 

You can keep a sepratre readme file and provide the link to that readme file here

Reply:
Updated the validations.md and also readme with override function